dotnet/targets/Xamarin.Shared.Sdk.targets

```
@@ -1022,6 +1048,39 @@
 		</ItemGroup>
 	</Target>
 
+	<Target Name="_CreateAOTDedupAssembly"
+			Condition="'$(_RunAotCompiler)' == 'true'">
```

Adding `Inputs` and `Outputs` makes it so that the assembly isn't recreated on every build, only the first build:

```suggestion
			Condition="'$(_RunAotCompiler)' == 'true'"
			Inputs="$(MSBuildThisFileFullPath)"
			Outputs="$(_DedupAssembly)"
			>
```

This also means you'll have to define `_DedupAssembly` elsewhere (maybe the `_ComputeVariables` target?)